database/src/main/resources/db/migration/v85__fribeløp_månedsberegning.sql

```diff
@@ -0,0 +1,34 @@
+with eps_info as (select (behandlingsinformasjon -> 'ektefelle' ->> 'alder')::int                        epsÅlder,
+                         behandlingsinformasjon -> 'bosituasjon' ->> 'ektemakeEllerSamboerUførFlyktning' epsErUfør,
+                         beregning,
+                         id
+                  from behandling),
+
+     mapped_fribeløpsverdier as (
```

Comment:
Hadde det ikke vært både enklere, tryggere og mer korrekt om vi utledet hvilken verdi som skal patches inn basert på hvilken fradragsstrategi som er brukt for den enkelte beregningen? Verdiene i feltet 
`beregning->'fradragStrategyName' ('Enslig', 'EpsOver67', 'EpsUnder67ÅrOgUførFlyktning', 'EpsUnder67År') kan oversettes direkte til beløpet vi ønsker oss, uten å måtte styre med alderen til eps.

Reply:
Det borde egentligen ge samma resultat, men ja er nog bedre og tryggare å bruke fradrag strategy navnet! Det er ju ändå den som avgör vilken typ av beregning som har blitt gjort. Byter. :)